tests/db/test_v202208190_to_v202209160.py

```diff
@@ -0,0 +1,41 @@
+"""
+    Copyright 2022 Inmanta
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Contact: code@inmanta.com
+"""
+import os
+from collections import abc
+
+import pytest
+
+
+@pytest.mark.db_restore_dump(os.path.join(os.path.dirname(__file__), "dumps", "v202208180.sql"))
```

Comment:
In #4829 the latest version of the database schema changed to v202209090. I think we need to test this change from that version and generate a new database dump.

Reply:
Hmm. The database dump created in #4829 seems to have an invalid version in its name. Do you fix that?

Reply:
Done!